src/sql/pg/create.sql

```diff
@@ -24,14 +24,33 @@ CREATE INDEX "L_XIE_FINGERPRINT" ON "L_ACCOUNTS"
   ("FINGERPRINT");
 
 
+CREATE TABLE IF NOT EXISTS "L_LU_REJECTION_REASON" (
+  "REJECTION_REASON_ID"   INTEGER NOT NULL,
+  "NAME"                  CHARACTER VARYING(10) NOT NULL,
+  "DESCRIPTION"           CHARACTER VARYING(255) NULL
+);
+
+CREATE INDEX "L_XPK_LU_TRANSFERS_REJECTION_R" ON "L_LU_REJECTION_REASON"
+  ("REJECTION_REASON_ID" ASC);
+ALTER TABLE "L_LU_REJECTION_REASON" ADD CONSTRAINT
+  "L_PK_LU_TRANSFERS_REJECTION_RE"
```

Comment:
OK, I guess that makes sense. So the convention is to use the table name after the index/constraint name and just truncate at 30 chars.